### DISCUSSION TITLE
unable to connect

### DISCUSSION THREAD

**Discussion by jonydebu** [gicamm/homeassistant-comelit#11] [General]
Hi guy, nice work.
I try to execute the mosquitto client and look at the output, but no answer .

Need some help if you can . Thanks in advance.

**Comment by iw6ovz** [comment]
Idem for me! I don't find the serial!

**Comment by gicamm** [comment]
I'm sorry for the considerable delay.
Something wrong happened with notifications.

To get the serial, you have to use the comelit application.
Steps are:

- Open the Comelit app
- Scan for a new hub device
- Copy the serial

Then use the serial number in configuration.yaml file.

Did you look at the [wiki](https://github.com/gicamm/homeassistant-comelit/wiki)?
Please let me know if I can help you.

**Comment by gicamm** [comment]
Hi,
You can use the Comelit app to discover the SN.
More details [here](https://github.com/gicamm/homeassistant-comelit/wiki#how-to-find-the-hub-serial)
Please let me know if I can help you.
